auslib/web/admin/swagger/api.yaml

```diff
@@ -1231,6 +1231,36 @@ paths:
                   type: string
                 example: ["releng", "qa"]
 
+  /roles:
+    get:
+      summary: Returns all of Roles in Balrog's DB
+      description: >
+        Returns all of the roles present in Balrog. [Docs](http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#id20)
+      tags:
+        - Roles
+      operationId: auslib.web.admin.views.mapper.roles_get
+      consumes: []
+      produces:
+        - application/json
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#id19"
+        description: "Returns all of Roles in Balrog's DB."
+      responses:
+        '200':
+          description: Get all Roles
+          schema:
+            type: object
+            properties:
+              users:
+                description: list of roles
+                type: array
+                uniqueItems: true
+                minItems: 0
+                items:
+                  type: string
+                example: ["releng", "qa"]
```

Comment:
Unfortunately, it's not safe to return a JSON list (https://haacked.com/archive/2008/11/20/anatomy-of-a-subtle-json-vulnerability.aspx/)